combat-trainer.lic

```diff
@@ -904,6 +905,24 @@ class SafetyProcess
 
   private
 
+  def recover_items(game_state)
+    return unless Flags['ct-germshieldlost']
+    get_item_wear_item(Flags['ct-germshieldlost'].first)
+    Flags.reset['ct-germshieldlost']
+  end
+
+  def get_item_wear_item(game_state, item)
+    return unless item
+    game_state.sheath_whirlwind_offhand
+    case bput("get #{item}", 'You pick up', 'What were you')
+    when 'What were you'
+      DRC.message "YOU HAVE DROPPED YOUR SHIELD AND WE CANNOT RECOVER!"
+    when 'You pick up'
+      case bput('wear my #{item}', 'You slide', 'You sling')
+    end
```

Comment:
I suggest using DRCI utility for getting the item and EquipmentManager for wearing it based on player's yaml config, and using the item variable in the error message rather than assume it'll always be a shield based on the generic nature of this method.

```suggestion
    if DRCI.get_item(item)
      @equipment_manager.wear_item?(@equipment_manager.item_by_desc(item))
    else
      DRC.message("YOU HAVE DROPPED YOUR #{item} AND WE CANNOT RECOVER!")
    end
```

Reply:
Is it ever not a shield that is lost? If not maybe this is being overcomplicated, the item at the feet will always be a shield and we simply need to get it and wear it, same as stowing ammo, etc. 

Do you want me to create a new utility method to handle getting something at the feet?

Reply:
It might be occurring on a thrown character, if this is the case, it complicates the logic.

Reply:
Confirmed it doesn't, he just hadn't noticed it before hand.